docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -409,7 +419,50 @@ load:
 
 .. code-block:: properties
 
-    exchange.base-directories=s3n://exchange-spooling-bucket-1,s3n://exchange-spooling-bucket-2
+    exchange.base-directories=s3://exchange-spooling-bucket-1,s3://exchange-spooling-bucket-2
+
+Google Cloud Storage
+~~~~~~~~~~~~~~~~~~~~
+
+If using Google Cloud Storage (GCS), it is recommended to configure a bucket lifecycle
```

Comment:
I know it is not common everywhere, but should we write new docs following the Google docs guide.
One thing I remember from there was that docs should directly address reader. 
So it should be sth like:
```
If you are using .... it is recommended you configure ...
```
@mosabua ?

Reply:
Yeah, second person is ideal. I don't think we need to be incredibly strict about it, but this would be an improvement.